classes/Renderer/Header.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+namespace MySociety\TheyWorkForYou\Renderer;
+
+/**
+ * Template Header Renderer
+ *
+ * Prepares variables for inclusion in a template header.
+ */
+
+class Header
+{
+
+    public $top_highlight;
+    public $nav_highlights;
+
+    /**
+     * Generate header data
+     *
+     * @param array  $data     An associative array of data to be made available to the template.
+     */
+
+    public function get_data($data = array())
```

Comment:
But it could use a private $data rather than passing it to all the sub-functions. Could do it all in the constructor in fact, given that's its only real purpose.